python/parquet_tools/csv2pq_utils.py

```diff
@@ -0,0 +1,111 @@
+# *****************************************************************************
+# *                       c s v 2 p q _ u t i l s . p y                       *
+# *****************************************************************************
+#
+# This file is part of parquet_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (http://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+The csv2pq_utils.py module contains utility functions used by the csv2pq
+command and its assiciated python modules.
+
+This module provides various functions to make life somewhat easier:
+ePrint    - print a message to stderr.
+Fatal     - print a numbered message to stderr and exit.
+getVal    - get the next value from the command line argument vector.
+traceBack - print a stack traceback.
+"""
+
+from __future__ import print_function
+
+import sys
+import traceback
+
+
+# *****************************************************************************
+# *                                e P r i n t                                *
+# *****************************************************************************
+
+def ePrint(*args, **kwargs):
+    "Print to stderr."
+
+    print(*args, file=sys.stderr, **kwargs)
+
+
+# *****************************************************************************
+# *                             t r a c e B a c k                             *
+# *****************************************************************************
+
+def traceBack():
```

Comment:
Not really. I have that in case I really want a traceback. You are living in the 60's where a command gave you a core dump as output when an error was encountered. Humans just want an error message not a traceback as they can't understand the traceback and it hides the error message. Whatever happened to human factors?

Reply:
We all robots at this point, not humans 🤖 Frankly, if you want to produce nice informational message for humans, I have no problem with that. I only complain because current approach is to only print exception text, which in my experience is rarely give user any useful info (and it also strips info that could be useful for developers).